src/server/ExpressTools.ts

```diff
@@ -95,16 +95,30 @@ export default class ExpressTools {
       // Create the options
       const options: https.ServerOptions = {};
       // Set the keys
-      options.key = fs.readFileSync(serverConfig['ssl-key']);
-      options.cert = fs.readFileSync(serverConfig['ssl-cert']);
+      // FIXME read certificates directly from config.json file. In the future: config for OICP in default tenant
+      if (serverConfig['ssl-key'].includes('-----BEGIN PRIVATE KEY-----')) {
+        options.key = serverConfig['ssl-key'];
+        options.cert = serverConfig['ssl-cert'];
+      } else {
+        options.key = fs.readFileSync(serverConfig['ssl-key']);
```

Comment:
the current implementation reads the certificate from the file system.
I kept that in to avoid conflicts.

But if you are sure that _options.key = fs.readFileSync(serverConfig['ssl-key']);_ is no longer needed, I will remove it

Reply:
If this is provided in the config.json / user provided service, we don't need this code anymore

Reply:
okay, I changed it